projects/cds/src/merchandising/facade/cds-merchandising-user-context.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
+import {
+  ConverterService,
+  ProductSearchPage,
+  ProductSearchService,
+  RouterState,
+  RoutingService,
+} from '@spartacus/core';
+import { combineLatest, Observable } from 'rxjs';
+import { distinctUntilChanged, map } from 'rxjs/operators';
+import {
+  MERCHANDISING_FACET_NORMALIZER,
+  MERCHANDISING_FACET_TO_QUERYPARAM_NORMALIZER,
+} from '../connectors/strategy/converters';
+import { MerchandisingUserContext } from '../model/merchandising-user-context.model';
+
+function isSameUserContext(
```

Comment:
Make this a `private` method instead of a function.

Reply:
Removed as it is no longer required due to the suggested change below to `cds-merchandising-user-context.service.ts` (I have gone a step further and made each stream return just a string of the property in the context object that they provide, so `distinctUntilChanged` should be able to compare those values with no extra functions)